iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndVectorizeLinalgTensorOps.cpp

```diff
@@ -183,27 +183,6 @@ void LLVMCPUTileAndVectorizePass::runOnOperation() {
       return signalPassFailure();
     }
   }
-  //
-  // Hosit hierarchical tiling indexing and other loop invariant transfer
-  // ops computation.
-  //
-  // Programmatic controlled lowering of vector.transfer only.
-  {
-    VectorTransferToSCFOptions vectorToSCFOptions =
```

Comment:
I think it is fine to remove those from here, not sure why they were done this early though.

Reply:
I think it was just a "copy" of the buffer pipeline.

Reply:
IIRU we might still need to vector -> scf + unroll part. Lets kick in benchmarks and land this if there is no regression.